src/frontend/apps/.cookiecutter/{{cookiecutter.app_name}}/data/queries/index.spec.tsx

```diff
@@ -0,0 +1,270 @@
+import fetchMock from 'fetch-mock';
+import React from 'react';
+import { QueryClient, QueryClientProvider, setLogger } from 'react-query';
+import { renderHook, WrapperComponent } from '@testing-library/react-hooks';
+
+import { {{cookiecutter.model_lower}}MockFactory } from 'apps/{{cookiecutter.app_name}}/utils/tests/factories';
+
+import {
+  use{{cookiecutter.model}},
+  use{{cookiecutter.model_plural}},
+  useCreate{{cookiecutter.model}},
+  useUpdate{{cookiecutter.model}},
+} from '.';
+
+setLogger({
+  // tslint:disable-next-line:no-console
+  log: console.log,
+  warn: console.warn,
+  // no more errors on the console
+  error: () => {},
+});
```

Comment:
Should not be usefull anymore

Reply:
It is for requests testing.

Reply:
@lunika added it in the test configuration, this should not be needed here